### PR TITLE
hybridcompute - update readme.md for SDK commands generation

### PR DESCRIPTION
**Pull request by yaotongms**
When releasing SDK for different languages, there are some API operations that we don't want to release to customers. 

This PR adds updates in readme to make sure that what gets released aligns with our .NET SDK.

### CODE REVIEW

**Review by rkmanda — APPROVED** [review]
:shipit:

**Review by weidongxu-microsoft — APPROVED** [review]

File: specification/hybridcompute/resource-manager/azure/telemetry.txt
Comment (weidongxu-microsoft) [inline specification/hybridcompute/resource-manager/azure/telemetry.txt:1]: I don't think we should check-in these files under azure folder.

File: specification/hybridcompute/resource-manager/readme.md [hunk 1]
```diff
@@ -138,32 +134,11 @@ directive:
         "default": {
           "description": "Error response describing why the operation failed.",
           "schema": {
-            "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            "$ref": "https://github.com/Azure/azure-rest-api-specs/blob/f6278b35fb38d62aadb7a4327a876544d5d7e1e4/specification/common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
```
Comment (weidongxu-microsoft) [inline specification/hybridcompute/resource-manager/readme.md:137]: why...

**Comment by yaotongms** [comment]
https://github.com/Azure/sdk-release-request/issues/5041

**Pull request merged**